File: Linguagem-Java/resolucao/soma_impares.java
import java.util.Locale;
import java.util.Scanner;

public class soma_impares {
  public static void main(String[] args) {
    Locale.setDefault(Locale.US);
    Scanner sc = new Scanner(System.in);

    // declaracao das variaveis a serem utilizadas na resolucao //
    int x, y, troca, soma;

    // input dos valores necessarios para a resolucao //
    System.out.println("Digite dois numeros:");
    x = sc.nextInt();
    y = sc.nextInt();
    
    // para garantir que seja analisado na ordem crescente //
    if (x > y) {
      troca = x;
      x = y;
      y = troca;
    }

    // criacao da regra for para percorrer a estrutura acima somando os valores impares //
    soma = 0;
    for (int i = x + 1; i < y; i++) {
      if (i % 2 != 0) {
        soma = soma + i;
      }
    }

    // outpur da mensagem final //
    System.out.println("Soma dos impares = " + soma);
    sc.close();
  }
}

File: Linguagem-C#/resolucao/soma_impares.cs
using System;
using System.Globalization;

namespace soma_impares
{
  class Program
  {
    static void Main(string[] args)
    {
      // declaracao das variaveis a serem utilizadas //
      int x, y, troca, soma;

      // imput dos dados para a resolucao do exercicio //
      Console.WriteLine("Digite dois numeros:");
      x = int.Parse(Console.ReadLine());
      y = int.Parse(Console.ReadLine());

      // criacao da logica para a soma dos numeros impares //
      if (x > y) {
        troca = x;
        x = y;
        y = troca;
      }

      soma = 0;
      for (int i = x+1; i < y; i++) {
        if (i % 2 != 0) {
          soma = soma + i;
        }
      }
      // output dos dados calculados acima //
      Console.WriteLine("Soma dos impares = " + soma);
    }
  }
}
